### DISCUSSION TITLE
Can you open a donate link ?

### DISCUSSION THREAD

**Discussion by vihangasa14** [lllyasviel/stable-diffusion-webui-forge#973] [General]
I would like to donate $ for hard work for forge and I want to forge live cause I don't have time learn comfyui. Would you please open buy me coffee or something like that other people will donte your hard work. @lllyasviel

**Comment by Kroke82** [comment]
Ouuu yeha, good idea, lllyasviel a lot of work hard, congratuletions

**Comment by V-Lex** [comment]
I'm waiting for that 👍👍👍
